pypxlib/__init__.py

```diff
@@ -24,12 +25,27 @@ def next(self):
 		# Python 3:
 		__next__ = next
 
-	def __init__(self, file_path, encoding='cp850'):
+	def __init__(self, file_path, blob_file_path=None, encoding='cp850'):
 		self.file_path = file_path
+		self.blob_file_path = blob_file_path
 		self.encoding = encoding
 		self.pxdoc = PX_new()
 		if PX_open_file(self.pxdoc, file_path.encode(self.PX_ENCODING)) != 0:
 			raise PXError('Could not open file %s.' % self.file_path)
+
+		# BLOB Support
+		# http://pxlib.sourceforge.net/documentation.php?manpage=PX_set_blob_file
+		if not blob_file_path:
+			# If not blob_file_path given, we try to acces automatically the related MB file
+			possible_blob_file = str(file_path).replace('.db', '.mb').replace('.DB', '.MB')
+			if os.path.isfile(possible_blob_file):
+				blob_file_path = possible_blob_file
+				self.blob_file_path = blob_file_path
+
+		# If given blob_file_path or file was automatically found we try to open it
+		if blob_file_path and PX_set_blob_file(self.pxdoc, blob_file_path.encode(self.PX_ENCODING)) < 0:
```

Comment:
The line goes over the character limit I normally use (80 chars, assuming tab is 4 spaces). I would split it as follows:

```python
if blob_file_path:
	blob_file_path_enc = blob_file_path.encode(self.PX_ENCODING)
	if PX_set_blob_file(self.pxdoc, blob_file_path_enc) < 0:
		raise PXError('Could not open BLOB file %s.' % self.blob_file_path)
```